base_import_async/data/queue_job_function_data.xml

```diff
@@ -0,0 +1,19 @@
+<odoo noupdate="1">
+    <record id="job_function_base_import_import_split_file" model="queue.job.function">
+        <field name="name"><![CDATA[<base_import.import>._split_file]]></field>
+        <field
+            name="related_action"
+            eval='{"func_name": "_related_action_attachment"}'
+        />
+    </record>
+    <record
+        id="job_function_base_import_import_import_one_chunk"
+        model="queue.job.function"
+    >
+        <field name="name"><![CDATA[<base_import.import>._import_one_chunk]]></field>
+        <field
```

Comment:
@guewen Wouldn't it be easier to have two fields: model and method_name and to compute the name from these two fields? This would allow a more natural syntax.

Reply:
Yes for sure! I add this to the remaining tasks

Reply:
Added in https://github.com/OCA/queue/pull/274/commits/27687b2ed9009edfd7f4baeb4dce0ebcdf025901